src/components/sections/HeroSection.tsx

```diff
@@ -10,7 +11,7 @@ const HeroSection = () => {
     backgroundRepeat: "no-repeat",
   }
   const backgroundStyle = {
-    backgroundImage: "/hero/HeroClouds.png",
+    backgroundImage: `url(${Clouds})`,
```

Comment:
okay, if the path was not working, then name the images with lowercase, so we know it's not a component